src/policy/fees_util.h

```diff
@@ -1,12 +1,14 @@
 // Copyright (c) 2024 The Bitcoin Core developers
 // Distributed under the MIT software license. See the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
-
```

Comment:
In  [fees]: add function that computes ancestors and descendants of txs:

I think this emptyline can stay?

Reply:
Added back